xds/src/main/java/io/grpc/xds/client/ControlPlaneClient.java

```diff
@@ -59,7 +59,8 @@
  * Common base type for XdsClient implementations, which encapsulates the layer abstraction of
  * the xDS RPC stream.
  */
-final class ControlPlaneClient {
+@Internal
+public final class ControlPlaneClient {
```

Comment:
It looks like this is public for a single line in GrpcXdsClientImplTestBase. We should just replace the `RpcRetryTask.class.getSimpleName()` with a string literal. (That seems like a code smell that we need to rearrange tests or whatnot, but that replacing it with a string is trivial for right now.)

Ditto for the LoadReportClient tasks.